crates/uv-resolver/src/requires_python.rs

```diff
@@ -711,6 +723,7 @@ mod tests {
     use std::cmp::Ordering;
     use std::collections::Bound;
     use std::str::FromStr;
+    use test_case::test_case;
```

Comment:
Can you move this to the group of non-std crate-external imports below? With `uv_pep440` and `uv_distribution_filename`.

The general pattern in the ecosystem is to put imports in three groups, in this order: std imports, non-`crate` imports, `crate` imports. (I also usually separate `core` and `alloc` imports too, but that's only for `no_std` libraries.)